Exercise-HTML-CSS-1/style.css

```diff
@@ -0,0 +1,105 @@
+
+#top{
```

Comment:
As I mention before, try to remove all the rules with id's and replace them with classes